src/query/functions/temporal/base.go

```diff
@@ -315,8 +337,8 @@ func (c *baseNode) processCompletedBlocks(
 // the datapoint list.
 func getIndices(
 	dps []ts.Datapoint,
-	lBound time.Time,
-	rBound time.Time,
+	lBound int64,
+	rBound int64,
```

Comment:
Can we use `xtime.UnixNano` here too? https://github.com/m3db/m3/blob/master/src/x/time/unix_nano.go#L27